templates/index.html

```diff
@@ -223,6 +223,15 @@ <h2 class="sponsors__title">Bronze Sponsors</h2>
                 </a>
             </div>
         </div>
+        <div class="sponsors sponsors--infrastructure">
+            <h2 class="sponsors__title">Infrastructure Sponsors</h2>
+            <div class="sponsors__content">
+                <a class="sponsors__link" href="https://vercel.com?utm_source=[bevy]&utm_campaign=oss">
+                    <img class="sponsors__logo" src="assets/vercel/logotype/light/vercel-logotype-light.png"
```

Comment:
Given that a SVG is available, I think it's better to use that, it weights less and gives better quality.
Also, what about putting it on a `sponsors` folder? We could move the other ones there also (in this PR, on another one).

```suggestion
                    <img class="sponsors__logo" src="assets/sponsors/vercel.svg"
```

Reply:
Related to this, I would remove all other Vercel logos/icons.

Reply:
Yeah, good point, I just threw everything in there and didn't think much of it. I'll remove it. I'll also create a sponsors folder for all those assets to make it a bit more organized.